.circleci/config.yml

```diff
@@ -293,8 +293,11 @@ jobs:
     working_directory: /home/circleci/project
     steps:
       - checkout
+      - run: cd flowmachine && sudo apt-get install -y graphviz && pipenv run pip install pygraphviz
       - restore_cache:
           key: flowmachine-deps-1-{{ checksum "flowmachine/Pipfile.lock" }}
+      # Need to install graphviz and pygraphviz manually because it was removed from the Pipfile
+      # (see https://github.com/Flowminder/FlowKit/issues/952)
       - run: cd flowmachine && pipenv install --dev --deploy && pipenv run pip install -e .
```

Comment:
I think this is why the test is failing - there's already a cache for this iteration of the lock file, so the install of pygraphviz isn't getting cached.

Reply:
Sure, that's why I moved the installation command outside of the save_cache/restore_cache bracket. But I clearly don't understand how CircleCI caching works. ;) Should I have moved it _after_ the save_cache command?

Reply:
You could bump the deps version, or wait until #954 is merged which will change the lock file so the cache gets made again